
class ExceptionHandler:
    def __init__(self) -> None:
        pass
